src/components/InngestCommHandler.ts

```diff
@@ -17,6 +17,7 @@ import { strBoolean } from "../helpers/scalar";
 import { createStream } from "../helpers/stream";
 import { stringifyUnknown } from "../helpers/strings";
 import { type MaybePromise } from "../helpers/types";
+import { type Logger } from "../middleware/logger";
```

Comment:
You're manually reordering imports to make ESLint happy? Have you used the Prettier plugin https://github.com/IanVS/prettier-plugin-sort-imports?

Reply:
> You're manually reordering imports to make ESLint happy?

@goodoldneon Oh heck no - all done automatically on save, but because of some creative use of `tsconfig.json` files for testing against different TS versions, sometimes automatically-generated imports were being added as `inngest/my/import` instead of relative imports.

Think it's sorted now, but just checking. :D

Reply:
I add the line manually but something that's running in `yarn dev` keep warning me to change it to a `type` and I just didn't bother to fight it.